src/test/java/com/sugarcrm/candybean/automation/webdriver/WebDriverSystemTest.java

```diff
@@ -36,13 +36,15 @@
 import com.sugarcrm.candybean.automation.element.Hook.Strategy;
 import com.sugarcrm.candybean.exceptions.CandybeanException;
 import com.sugarcrm.candybean.runner.VRunner;
-import org.openqa.selenium.WebDriver;
 
 @RunWith(VRunner.class)
 public class WebDriverSystemTest {
 
 	private WebDriverInterface iface;
-	
+
+	final String testPage1 = "file://"+ System.getProperty("user.dir")+"/resources/html/test/testPlayground.html";
+	final String testPage2 = "file://"+ System.getProperty("user.dir")+"/resources/html/test/onOffScreen.html";
```

Comment:
`testPage1` and `testPage2` are not descriptive names. `testPlaygroundPage` and `onOffScreenPage` are better names imo.